var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -39,12 +42,20 @@ class Qt(Package):
     # depends_on("flex")
     # depends_on("bison")
     # depends_on("ruby")
-    # depends_on("icu4c")
+    depends_on("icu4c")
 
     # OpenGL hardware acceleration
-    depends_on("mesa", when='@4:+mesa')
-    depends_on("libxcb")
-
+    #depends_on("mesa", when='@4:+mesa')
+    if "darwin" in sys.platform:
+        pass
+
+    elif "linux" in sys.platform:
+        depends_on("libxcb")
```

Comment:
`libxcb` seems to be not self-contained https://github.com/LLNL/spack/issues/863 . But that's another issue. Just saying that it will probably fail on anything but `ubuntu` with those packages installed manually.